opennms-doc/guide-admin/src/asciidoc/text/minion/kafka.adoc

```diff
@@ -14,7 +14,7 @@ This means that _ActiveMQ_ is still required even when _Kafka_ support is enable
 
 _Kafka_ must be enabled on both _{opennms-product-name}_ and _Minion_ to function.
 
-The _Kafka_ server must be compatible with _Kafka_ client version `0.10.1.1_1`.
+The _Kafka_ server must be compatible with _Kafka_ client version `1.0.1`.
```

Comment:
The links to the producer and consumer configuration references should also be updated.

Reply:
Ah, somehow I was mistaken that they were updated.